lib/react_native_loader.js

```diff
@@ -0,0 +1,24 @@
+var util = require('./util');
+
+// browser specific modules
+util.crypto.lib = require('crypto-browserify');
```

Comment:
Does this get resolved to the pure-js version used in browsers or something specific to react native?